tests/test_markup.py

```diff
@@ -177,6 +177,16 @@ def test_load_system(
         assert value in entity.entity.value["value"]
 
 
+def test_entity_values(query_factory):
+    processed_query = markup.load_query(
+        "lets meet {2 hours from now|sys_interval}", query_factory, query_options={"timestamp": 1592002800}
```

Comment:
we should add more tests to cover the cases: "from ... to ...", "...to..."

Reply:
This is a confusing test. "2 hours from now" is a sys_time, while "from now" is a sys_interval. I think better would be tests that explicitly test each one-sided interval, plus the bounded interval:
 - "let's meet after 3:00"
 - "let's meet until 3:00"
 - "let's meet from now until 3:00"